javaagent-api/src/main/java/io/opentelemetry/javaagent/instrumentation/api/internal/ClassLoaderMatcherCacheHolder.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.javaagent.bootstrap;
+package io.opentelemetry.javaagent.instrumentation.api.internal;
```

Comment:
It still lives in bootstrap classloader, right?

Reply:
Yes - but at least it's not in the (internal) `javaagent-bootstrap` module.

Reply:
Why exactly do we consider bootstrap module as "internal"?

Reply:
There've been mentions of making it internal: https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/2779#discussioncomment-601705